twitter_nlp/core.py

```diff
@@ -1,10 +1,69 @@
-def cool_add(a, b):
-    return(a + b)
+"""This module enables a political sentiment analysis of user tweets.
 
+This module uses natural language processing to determine the sentiment
+of twitters users. A pre-trained model evaluates recent tweets and 
+provides a summary of measures of interest (e.g., political sentiment).
+The model has been trained with tweets categorised as conservative vs. 
+liberal. Tweets are accessed via the twitter API.
 
-def ploty():
-    import numpy as np
-    import matplotlib.pyplot as plt
-    x = np.linspace(0, 20, 100)
-    plt.plot(x, np.sin(x))
-    plt.show()
+Example:
+    $ python core.py
+
+Functions:
+    simple_nlp(): Counts nouns, adjectives, and verbs of tweets.
+
+Todo:
+    * incorporate tweet generator by using twitter API package
+    * incorpoarte full sentiment analysis
+"""
+import nltk
+from nltk.corpus import twitter_samples
+from nltk.tag import pos_tag_sents
+
+class nlp:
```

Comment:
Classes in python should be in PascalCase (i.e., starting with a capital letter). In general, abbreviations should be in all caps, so you can consider changing the name of this class to `NLP` (all capital letters),